packages/taco/test/conditions/context.test.ts

```diff
@@ -501,17 +475,17 @@ describe('context', () => {
       it('accepts on a hard-coded parameter', async () => {
         const customContractCondition = new ContractCondition({
           ...contractConditionObj,
-          parameters: [USER_ADDRESS_PARAM_EIP1271, 100],
+          parameters: [USER_ADDRESS_PARAM_DEFAULT, 100],
         });
         const conditionContext = new ConditionContext(customContractCondition);
         conditionContext.addAuthProvider(
-          USER_ADDRESS_PARAM_EIP1271,
-          authProviders[USER_ADDRESS_PARAM_EIP1271],
+          USER_ADDRESS_PARAM_DEFAULT,
+          authProviders["EIP4361"],
```

Comment:
This original goal was for variety of the tests., so that we aren't just using EIP4361 all the time.

```suggestion
          authProviders["EIP1271"],
```